backend/routes_chemicals.py

```
@@ -191,86 +206,104 @@ def chemical_barcode_route(id):
 
     # Issue a chemical
     @app.route('/api/chemicals/<int:id>/issue', methods=['POST'])
+    @secure_login_required
```

Ruff (0.11.9)

Undefined name `secure_login_required`

(F821)